common/src/jni/main/cpp/conscrypt/native_crypto.cpp

```diff
@@ -2592,8 +2590,9 @@ static void NativeCrypto_EVP_DigestUpdate(JNIEnv* env, jclass, jobject evpMdCtxR
 // EVP_DigestSignUpdate and EVP_DigestVerifyUpdate are functions in BoringSSl but not in OpenSSL.
 // The reason for the two wrapper functions below is that we need a function pointer which can be
 // provided to evpUpdate.
-// TODO: Remove these two wrapper functions once Conscrypt no longer supports OpenSSL or once
-// OpenSSL offers EVP_DigestSignUpdate and EVP_DigestVerifyUpdate as functions rather than macros.
+// TODO(nmittler): Remove these two wrapper functions once Conscrypt no longer supports
+// OpenSSL or once OpenSSL offers EVP_DigestSignUpdate and EVP_DigestVerifyUpdate as functions
+// rather than macros.
```

Comment:
Conscrypt no longer supports OpenSSL. You can just say "Remove these two wrapper functions" now and probably go do it.